app/Http/Controllers/Api/V1/ExpenseController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1;
+
+use App\Expense;
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+
+class ExpenseController extends Controller
+{
+    public function mostExpense()
+    {
+        $mostExpense = Expense::select(\DB::raw('MONTH(date) AS expenseMonth'), \DB::raw('SUM(amount) AS totalExpense'))
+                    ->whereYear('date', Carbon::now()->year)
+                    ->groupBy('expenseMonth')
+                    ->orderBy('totalExpense', 'DESC')
+                    ->first();
+
+        return response()->json($mostExpense);
```

Comment:
response jsonlarda data gönderirken şu formatta göndermen daha anlaşılır ve iyi olacaktır.

```suggestion
        return response()->json([
                'data' => $mostExpense,
        ]);
```